### PR TITLE
feat(#1601): add support for protocol parameter change governance action

### PR DESCRIPTION
**Pull request by MSzalowski**
## List of changes

- add support for protocol parameter change governance action

## Checklist

- [related issue](https://github.com/IntersectMBO/govtool/issues/1601)
- [x] My changes generate no new warnings
- [x] My code follows the [style guidelines](https://github.com/IntersectMBO/govtool/tree/main/docs/style-guides) of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the [changelog](https://github.com/IntersectMBO/govtool/blob/main/CHANGELOG.md)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: govtool/backend/sql/list-proposals.sql [hunk 1]
```diff
@@ -83,11 +90,16 @@ FROM
...
     LEFT JOIN voting_anchor ON voting_anchor.id = gov_action_proposal.voting_anchor_id
+    LEFT JOIN param_proposal as proposal_params ON gov_action_proposal.param_proposal = proposal_params.id
     LEFT JOIN off_chain_vote_data ON off_chain_vote_data.voting_anchor_id = voting_anchor.id
     LEFT JOIN off_chain_vote_gov_action_data ON off_chain_vote_gov_action_data.off_chain_vote_data_id = off_chain_vote_data.id
     LEFT JOIN voting_procedure ON voting_procedure.gov_action_proposal_id = gov_action_proposal.id
-    LEFT JOIN LatestDrepDistr ldd ON ldd.hash_id = voting_procedure.drep_voter
-        AND ldd.rn = 1
+    LEFT JOIN LatestDrepDistr ldd_drep ON ldd_drep.hash_id = voting_procedure.drep_voter
+        AND ldd_drep.rn = 1
+    LEFT JOIN LatestDrepDistr ldd_pool ON ldd_pool.hash_id = voting_procedure.pool_voter
```
Comment (Ryun1) [inline govtool/backend/sql/list-proposals.sql:99]: is counting SPO and CC votes meant to be a part of this PR?

Comment (MSzalowski) [reply govtool/backend/sql/list-proposals.sql:99]: Not on the UI side. Just added this while adding protocol params to the list proposals

Comment (Ryun1) [reply govtool/backend/sql/list-proposals.sql:99]: Okay cool

File: govtool/frontend/src/components/organisms/GovernanceActionDetailsCardData.tsx [hunk 2]
```diff
@@ -9,9 +12,43 @@ import {
...
+
+type TabPanelProps = {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+};
+
+const CustomTabPanel = ({ children, value, index }: TabPanelProps) =>
+  value === index && (
+    <Box sx={value === index && index === 1 ? { overflow: "scroll" } : {}}>
```
Comment (jdyczka) [inline govtool/frontend/src/components/organisms/GovernanceActionDetailsCardData.tsx:32]: `value === index` is checked twice, but can't all tabs have `overflow: "auto"`?

File: govtool/frontend/src/components/organisms/GovernanceActionDetailsCardData.tsx [hunk 3]
```diff
@@ -111,27 +194,58 @@ export const GovernanceActionDetailsCardData = ({
...
+                width: !isMobile ? "auto" : "50%",
+              }}
+            />
+            <StyledTab
+              data-testid="parameters-tab"
+              label="Parameters"
+              sx={{
+                textTransform: "none",
+                width: !isMobile ? "auto" : "50%",
+              }}
```
Comment (jdyczka) [inline govtool/frontend/src/components/organisms/GovernanceActionDetailsCardData.tsx:226]: `StyledTab` already has `textTransform: "none"`, maybe move `width` there as well?

**Review by jdyczka — APPROVED** [review]

**Pull request merged**